scripts/copycat_mode_quit.sh

```diff
@@ -4,11 +4,35 @@ CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 source "$CURRENT_DIR/helpers.sh"
 
+unbind_cancel_bindings() {
+	local cancel_mode_bindings=$(copycat_quit_copy_mode_keys)
+	local key
+	for key in $cancel_mode_bindings; do
+		tmux unbind-key -n "$key"
+	done
+}
+
+unbind_prev_next_bindings() {
+	tmux unbind-key -n "$(copycat_next_key)"
+	tmux unbind-key -n "$(copycat_prev_key)"
+}
+
+unbind_all_bindings() {
+	grep -v copycat <"${TMPDIR:-/tmp}/copycat_$(whoami)_recover_keys" | while read -r key_cmd; do
+		sh -c "tmux $key_cmd"
```

Comment:
This line was the fix for #121 (original introduced in 5f6bfc06b371329e6ca90adbbd83cdfa554828aa), but https://github.com/tmux-plugins/tmux-copycat/commit/6bcd4dc4eb3e9f8106aa03f422b54e6dd19f4044 removed it.